management/server/src/main/java/org/apache/karaf/management/KarafMBeanServerGuard.java

```diff
@@ -346,11 +346,20 @@ private boolean canBypassRBAC(BulkRequestContext context, ObjectName objectName,
         return false;
     }
 
-    void handleInvoke(ObjectName objectName, String operationName, Object[] params, String[] signature) throws IOException {
-        handleInvoke(null, objectName, operationName, params, signature);
+    void handleInvoke(MBeanServer mbs, ObjectName objectName, String operationName, Object[] params, String[] signature) throws IOException, InstanceNotFoundException {
```

Comment:
Is it still backward compatible by callers?

Reply:
yes, all callers are covered. Package visibility seems to be either historic or for future unit tests.